### PR TITLE
F/add classes

### PR DESCRIPTION
**Pull request by Gnwin**
## OOP school library: add Person, Student and Teacher classes

In this project, classes for Person, Student and Teacher were added according to ruby style guidelines.
All requirements for this project have been met according to this requirement:

https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md

### CODE REVIEW

File: lib/person.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  private
+
+  def is_of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    is_of_age? || parent_permission
+  end
```
Comment (mikethreels) [inline lib/person.rb]: - [x] in this setup the `can_use_services?` method is also private please make sure this is a public method.

File: lib/person.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+end
+
+person = Person.new('Peter', 31)
+p person.parent_permission
```
Comment (mikethreels) [inline lib/person.rb]: - [x] please remove any code added to test the application.

File: lib/student.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "¯\(ツ)/¯"
+  end
+end
+
+student = Student.new('Peter', 31, 'ruby')
+p student
+p student.parent_permission
+p student.parent_permission = false
+p student.parent_permission
+p student.play_hooky
```
Comment (mikethreels) [inline lib/student.rb]: - [x] please remove any code added to test the application.

File: lib/teacher.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+  private
+
+  def can_use_services?
+    true
+  end
+end
+
+teacher = Teacher.new('Peter', 31, 'science')
+p teacher
```
Comment (mikethreels) [inline lib/teacher.rb]: - [x] please remove any code added to test the application.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Godwin, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: lib/person.rb (hunk 1 above)
Comment (Gnwin) [reply lib/person.rb]: done

**Review by mikethreels — APPROVED** [review]
Hi Godwin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**